conference/models.py

```diff
@@ -1000,7 +1000,7 @@ def events_score_by_attendance(self, conference):
         # Parallel obviously can not participate in both, so the
         # his vote should be scaled
         scores = defaultdict(lambda: 0.0)
-        for evt, users in events.items():
+        for evt, users in list(events.items()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;